json/ContributionCredential/1-0-0.json

```diff
@@ -57,7 +57,7 @@
           "type": "string"
         },
         "name": {
-          "title": "Name",
+          "title": "DAO Name",
```

Comment:
Do we want to limit this to DAOs as opposed to something like "Organization Name"?